app/frontend/src/components/ContributorForm/ContributorForm.tsx

```diff
@@ -218,61 +209,100 @@ export default function ContributorForm({ processForm }: ContributorFormProps) {
             className={classes.textbox}
           />
           <Controller
-            id="contributeWays"
+            id="contribute"
             control={control}
-            name="contributeWays"
+            name="contribute"
             defaultValue=""
-            render={({ onChange, value }) => (
+            render={({ onChange }) => (
               <FormControl>
-                <FormLabel className={classes.radioLabel}>
-                  {CONTRIBUTE_WAYS_LABEL}
+                <FormLabel component="legend" className={classes.radioLabel}>
+                  {CONTRIBUTE_LABEL}
                 </FormLabel>
-                <FormGroup>
-                  {CONTRIBUTE_WAYS_OPTIONS.map((option) => (
+                <RadioGroup
+                  className={classes.contributeRadio}
+                  name="contribute-radio"
+                  onChange={onChange}
+                >
+                  {CONTRIBUTE_OPTIONS.map((option) => (
                     <FormControlLabel
-                      key={option.name}
-                      value={option.name}
-                      control={
-                        <Checkbox
-                          checked={value.includes(option.name)}
-                          onChange={() =>
-                            toggleCheckbox(option.name, value, onChange)
-                          }
-                          name={option.name}
-                        />
-                      }
-                      label={option.description}
+                      key={option}
+                      value={option}
+                      control={<Radio />}
+                      label={option}
                     />
                   ))}
-                </FormGroup>
-                <FormHelperText error={!!errors?.contributeWays?.message}>
-                  {errors?.contributeWays?.message ?? " "}
-                </FormHelperText>
+                </RadioGroup>
               </FormControl>
             )}
           />
-          <Typography
-            variant="body1"
-            htmlFor="expertise"
-            component="label"
-            className={classes.label}
+          <Collapse
+            in={
+              watchContribute !== CONTRIBUTE_OPTIONS[2] &&
```

Comment:
```suggestion
              watchContribute !== "No" &&
```
I feel like in this case, this is clearer as opposed to having to check out what the third option is and an ok compromise to have the value in the file

Reply:
I don't think that will work once we have translations. I'd rather leave this. (I did `CONTRIBUTE_OPTIONS as const` so you can just hover to see the values)

Reply:
We'll probably have to change this again anyway when we do translations, but if hovering gives the value in VS Code then let's leave it 👍